### DISCUSSION TITLE
Issues starting my first plot

### DISCUSSION THREAD

**Discussion by Theamazinggenko** [ericaltendorf/plotman#790] [Q&A] [answered]
when i do "plotman plot" in terminal it produces this log file:

Starting plotting progress into temporary dirs: /media/test/New Volume and /media/test/New Volume
ID: -----------------------------------------------------
Plot size is: 32
Buffer size is: 4608MiB
Using 128 buckets
Using 4 threads of stripe size 65536
Caught plotting error: Could not open /media/test/New Volume/plot-k32-2021-06-14-21-58------------------------.plot.sort.tmp: No such file or directory.

what can i do to fix this? (not sure what info is sensitive so i have replaced with a -)

**Comment by racergoodwin** [comment, marked as answer]
Ensure there is a disk mounted to /media/test/New Volume
If there is, you could try putting "" around the folder in plotman.yaml, i.e. "/media/test/New Volume"
You could also  rename it to something without a whitespace and remount it, then change the plotman.yaml to reflect the new folder and restart plotman.

**Reply by Theamazinggenko** [reply]
that fixed it thanks
